MobiFlight/Board.cs

```diff
@@ -279,6 +322,36 @@ public class Board
         /// </summary>
         public UsbDriveSettings UsbDriveSettings;
 
+        /// <summary>
+        /// Base path for custom firmware
+        /// </summary>
+        public String BasePath;
```

Comment:
Will this be specified in the JSON? Or is it calculated? If possible I would try and calculate this based on where the config file is, or say it must be located side-by-side with the config file. Otherwise it's another path for the developer to specify and possibly get wrong.